alerting/provider/custom/custom_test.go

```diff
@@ -125,22 +125,27 @@ func TestAlertProvider_buildHTTPRequest(t *testing.T) {
 		{
 			AlertProvider: customAlertProvider,
 			Resolved:      true,
-			ExpectedURL:   "https://example.com/endpoint-group/endpoint-name?event=RESOLVED&description=alert-description&url=https://example.com",
-			ExpectedBody:  "endpoint-name,endpoint-group,alert-description,https://example.com,RESOLVED",
+			ExpectedURL:   "https://example.com/endpoint-group/endpoint-name?event=RESOLVED&description=alert-description&url=https://example.com&error=",
+			ExpectedBody:  "endpoint-name,endpoint-group,alert-description,https://example.com,RESOLVED,",
 		},
 		{
 			AlertProvider: customAlertProvider,
 			Resolved:      false,
-			ExpectedURL:   "https://example.com/endpoint-group/endpoint-name?event=TRIGGERED&description=alert-description&url=https://example.com",
-			ExpectedBody:  "endpoint-name,endpoint-group,alert-description,https://example.com,TRIGGERED",
+			ExpectedURL:   "https://example.com/endpoint-group/endpoint-name?event=TRIGGERED&description=alert-description&url=https://example.com&error=error",
+			ExpectedBody:  "endpoint-name,endpoint-group,alert-description,https://example.com,TRIGGERED,error",
 		},
 	}
 	for _, scenario := range scenarios {
 		t.Run(fmt.Sprintf("resolved-%v-with-default-placeholders", scenario.Resolved), func(t *testing.T) {
+			result := &endpoint.Result{Errors: []string{}}
+			if !scenario.Resolved {
+				result.Errors = append(result.Errors, "error")
+			}
```

Comment:
This is not a valid assumption, because a result that isn't successful doesn't necessarily have an error. 
Result errors are usually things that were truly not expected, such as a condition being invalid (as opposed to a condition failing) or a hostname being unresolvable/unreachable.